### PR TITLE
Add missing MiMa entries for 2.4

### PR DESCRIPTION
**Pull request by jrudolph**
First step for fixing mima in `release-2.4`.

Refs #22306.

### CODE REVIEW

File: project/MiMa.scala [hunk 1]
```diff
@@ -1162,7 +1160,15 @@ object MiMa extends AutoPlugin {
 
         // #21894 Programmatic configuration of the ActorSystem
         ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorSystemImpl.this")
-      ))
+      ),
+      "2.4.16" -> Seq(
+        // internal classes
+        FilterAnyProblemStartingWith("akka.remote.artery")
```
Comment (ktoso) [inline project/MiMa.scala]: 👍

**Review by ktoso — APPROVED** [review]
LGTM

**Review by johanandren — APPROVED** [review]
LGTM

**Comment by jrudolph** [comment]
Thanks for the quick reviews!

**Comment by jrudolph** [comment]
PLS BUILD

**Pull request merged**